Frontend/android-native/app/src/user/nammaYatri/assets/juspay/juspay_assets.json

```
@@ -373,6 +373,7 @@
       "ny_ic_drop_indicator" : true,
       "ny_ic_drop_loc_marker" : true,
       "ny_ic_drop_red_indicator" : true,
+      "ny_ic_earn_through_parcel" : true,
```

Biome (1.9.4)

[error] The key ny_ic_earn_through_parcel was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)